<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GPS</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(122, 122, 122);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>801</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Grafo</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>-30</x>
        <y>0</y>
        <width>831</width>
        <height>501</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout"/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>MAtriz Floyd</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>701</width>
        <height>411</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pbFloyd">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Floyd</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>40</y>
      <width>231</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Arista</string>
    </property>
    <widget class="QPushButton" name="pbAddArista">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Arista</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbDeleteArista">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete Arista</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbEditArista">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit Arista</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lePeso">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>130</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="leInicio">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="leFinal">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Nodo 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Nodo 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Peso</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>230</y>
      <width>231</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Look For..</string>
    </property>
    <widget class="QComboBox" name="cmbStart">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Place</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Destiny</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbDestiny">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Best Way to Go</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbGo">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>GO!</string>
     </property>
    </widget>
    <widget class="QListWidget" name="list">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>191</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="cmbDay">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Select a Day</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Monday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tuesday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Wednesday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Thursday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Friday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Saturday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sunday</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cmbHour">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Select an Hour</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0-1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1-2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2-3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3-4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4-5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5-6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6-7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7-8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8-9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9-10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10-11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11-12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12-13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13-14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14-15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15-16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16-17</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>17-18</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>18-19</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19-20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20-21</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>21-22</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>22-23</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>23-0</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pbSwitch_Graph">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Switch Graph</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1059</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>148</width>
     <height>365</height>
    </size>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="floating">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>Places</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <widget class="QListWidget" name="places">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>331</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
